src/pages/Products/ProductDetail/ProductDetail.js

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import Modal from './components/Modal';
+import ProductInfo from './components/ProductInfo';
+import './ProductDetail.scss';
+
+function ProductDetail() {
+  const navigate = useNavigate();
+
+  const [modal1, setModal1] = useState(false);
+  const [modal2, setModal2] = useState(false);
+  const [productInfo, setProductInfo] = useState([]);
+
+  const handleModal1 = e => {
+    setModal1(true);
+  };
+
+  const handleModal2 = e => {
+    setModal2(true);
+  };
+
+  const goToOrder = e => {
+    navigate('/order');
+  };
+
+  const goToCart = e => {
+    navigate('/cart');
+  };
```

Comment:
```suggestion
  const [productInfo, setProductInfo] = useState([]);
  const [currentModal, setCurrentModal] = useState('');

  const handleModal = e => {
    setCurrentModal(e.target.name);
  };

  const goToPage = path => {
    navigate(`/${path}`);
  };
```

모달 별로 state를 관리하기 보단 하나의 state로 관리할 수 있습니다.
그리고 페이지 이동하는 것도 인자로 넘겨주는 방식으로 구현할 수 있겠네요!

Reply:
모달 띄우는 로직이 변경돼서 ... 저 방식이 아닌 조건값을 넣어 다른 모달창을 띄워줘야합니다 ... ! 
아직 어려워서 구현은 못했어요 ㅜ_ㅜ

Reply:
오셔서 한번 같이 해보시죠!